hooks/node_handlers/backgroundimageplane_handler.py

```diff
@@ -0,0 +1,36 @@
+"""Hook class to handle Houdini 2D image read nodes.
+
+Designed to inherit ``base_import_handler`` i.e. define the ``hook`` attribute
+in your project configuration settings with the following inheritance:
+
+.. code-block:: yaml
+    :caption: env/includes/settings/tk-houdini_node_handlers.yml
+
+    node_handlers.asset_step:
+    - node_type: file
+      node_category: Cop2
```

Comment:
```suggestion
    - node_type: backgroundimageplane
      node_category: Object
```

Should these match the class attributes?

Reply:
weird, this is not what my vscode is showing me